packages/handlersjs-http/lib/handlers/http-handler-static-asset.spec.ts

```diff
@@ -45,12 +45,19 @@ describe('HttpHandlerStaticAssetService', () => {
 
   describe('handle()', () => {
 
-    it('throws an UnsupportedMediaTypeHttpError when no accept header is found', async() => {
+    it('should set Content-Type when Accept is undefined', async() => {
 
-      const response = lastValueFrom(service
-        .handle({ ...context, request: { ...context.request, headers: { accept: undefined } } }));
+      const contentType = 'text/plain';
+      const absolutePath = join(__dirname, '../../test/');
+      const absoluteService = new HttpHandlerStaticAssetService(absolutePath, contentType);
+
+      context.request.parameters.filename = 'test.txt';
+
+      const response = await lastValueFrom(absoluteService.handle(
+        { ...context, request: { ...context.request, headers: { accept: undefined } } }
+      ));
 
-      await expect(response).rejects.toThrowError(new UnsupportedMediaTypeHttpError('No accept header found'));
+      await expect(response.headers).toStrictEqual({ 'Content-Type': `${contentType}` });
 
     });
```

Comment:
@LiamThys 
I see the PR is merged already but I have some remarks on this test.

Try to:
 - Always use resolves / rejects, and avoid awaiting the function
 - Reuse existing variables, no need to create a new handler
 - Be wise with the use of `toStrictEqual`. If we ever decide to add a header this test will fail.
 - If you adapt a test (it) try to get rid of all red underlines, this way older/messy tests are cleaned up gradually. They might not error on linting but still.
 
 I see you probably took inspiration from the other tests in this file which appear to have been changed multiple times over the past months by different people and have become a bit messy, so no worries. Just wanted to give some tips.
 
 This is what I would have done:
 ```
     it('should handle a request when Accept is undefined', async() => {

      context.request.parameters!.filename = 'test.txt';
      delete context.request.headers.accept;

      const response = lastValueFrom(service.handle(context));

      await expect(response).resolves.toEqual(expect.objectContaining({
        headers: expect.objectContaining({
          'Content-Type': 'text/plain',
        }),
      }));

    });
```

Could you react an emoji so I know you have seen this comment, thanks.

Reply:
This review helps alot thanks